python/pyrogue/protocols/_uart.py

```diff
@@ -30,7 +30,11 @@ def __init__(self, device, baud, timeout=1, **kwargs):
         self._workerThread = threading.Thread(target=self._worker)
         self._workerThread.start()
 
+    # Deprecated
     def close(self):
+        self.stop()
```

Comment:
Need to add a print statement about how  `close()` is deprecated and will be removed in the future and should use `stop()` instead

Reply:
I think this will flush out over time and the calls to close() won't cause any issues.